daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Node.scala

```diff
@@ -81,6 +81,11 @@ object Node {
       chosenValue: Val,
       stakeholders: Set[Party],
       signatories: Set[Party],
+      /** Note that here we have both actors and controllers because we use this
+        * data structure _before_ we validate the transaction. However for every
+        * valid transaction the actors must be the same as the controllers. This
+        * is we removed the controllers field in transaction version 6.
```

Comment:
Maybe a missing "why" as in "why we removed"?

Reply:
done